api/lib/domain/models/Challenge.js

```diff
@@ -102,7 +102,7 @@ class Challenge {
   }
 
   hasSkill(searchedSkill) {
-    return this.skills.filter((skill) => skill.name === searchedSkill.name).length > 0;
+    return this.skills.filter((skill) => skill.id === searchedSkill.id).length > 0;
```

Comment:
return this.skills.some((skill) => skill.id === searchedSkill.id);

Reply:
Fait, merci